test/ui/data-models/list/DataModelListTable.test.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import 'test/enzyme-init';
+import { shallow } from 'enzyme';
+
+import DataModelListTable from 'ui/data-models/list/DataModelListTable';
+
+describe('DataModelListTable', () => {
+  let component;
+  beforeEach(() => {
+    component = shallow(<DataModelListTable page={1} pageSize={1} totalItems={1} />);
+  });
+  it('renders component without crashing', () => {
+    expect(component.exists()).toEqual(true);
+  });
+  it('root component has class DataModelListTable', () => {
+    expect(component.hasClass('DataModelListTable')).toEqual(true);
+  });
+  describe('inner table', () => {
+    it('renders table header with 4 cols', () => {
+      expect(component.find('table thead tr th')).toHaveLength(4);
+    });
+    it('renders table with correct structure', () => {
+      component = shallow(<DataModelListTable pageSize={1} page={1} totalItems={1} />);
+      expect(component.find('table thead tr').exists()).toEqual(true);
+      expect(component.find('table thead tr th').exists()).toEqual(true);
+    });
+
+    it('errors without a page', () => {
+      const consoleError = jest.spyOn(console, 'error').mockImplementation(() => {});
+      shallow(<DataModelListTable pageSize={1} totalItems={1} />);
+      expect(consoleError).toHaveBeenCalled();
+      consoleError.mockReset();
```

Comment:
There is a lot of duplicated code. Please put all the tests spying on console.error in their own  `describe`, and call `const consoleError = jest.spyOn(console, 'error').mockImplementation(() => {});` in a `beforeEach`, and `consoleError.mockReset(); consoleError.mockRestore();` in `afterEach`

Reply:
because only 3 tests require the consoleError mocking I'd argue that if we are going to use the beforeEach and afterEach methods we do put the 3 tests in their own describe.